packages/perseus/src/widgets/radio/__stories__/focus-ring.stories.jsx

```diff
@@ -1,38 +1,38 @@
 // @flow
 import * as React from "react";
 
+import * as styleConstants from "../../../styles/constants.js";
 import FocusRing from "../focus-ring.jsx";
 
-type StoryArgs = {||};
+type StoryArgs = {|
+    children: React.Node,
+    color: string,
+    visible: boolean,
+|};
 
 type Story = {|
     title: string,
+    args: StoryArgs,
 |};
 
 export default ({
     title: "Perseus/Widgets/Radio/Focus Ring",
+    args: {children: "", color: styleConstants.kaGreen, visible: true},
 }: Story);
 
 export const EmptyPropsObject = (args: StoryArgs): React.Node => {
-    return <FocusRing />;
-};
-
-export const VisibleSetToTrue = (args: StoryArgs): React.Node => {
-    return <FocusRing visible={true} />;
-};
-
-export const VisibleSetToFalse = (args: StoryArgs): React.Node => {
-    return <FocusRing visible={false} />;
-};
-
-export const ColorRed = (args: StoryArgs): React.Node => {
-    return <FocusRing color="red" />;
-};
-
-export const SingleSpanChild = (args: StoryArgs): React.Node => {
-    return (
-        <FocusRing>
-            <span>test</span>
-        </FocusRing>
-    );
```

Comment:
I didn't think these stories were anything but noise:

<img width="227" alt="Screen Shot 2022-10-05 at 3 58 08 PM" src="https://user-images.githubusercontent.com/16308368/194163126-a4b2af0e-fb57-465f-9f75-eab0f0e3087e.png">

The focus-ring also appears in other stories:

<img width="201" alt="Screen Shot 2022-10-05 at 3 58 40 PM" src="https://user-images.githubusercontent.com/16308368/194163245-7176b48f-3ad6-4665-a578-65e78a72cc7b.png">

Since it's such a small component, the existing stories didn't really say much about the component, and it shows up in other stories that are more descriptive of its use-case, I decided that having one interactive story was better than having many non-interactive stories in this case.

<img width="1009" alt="Screen Shot 2022-10-05 at 4 03 59 PM" src="https://user-images.githubusercontent.com/16308368/194163297-18a9a527-4cb1-40ca-834f-58d2e6e95a35.png">

Reply:
👍 Using storybook controls seems like a great alternative.